src/gpu/SDL_gpu.c

```diff
@@ -178,6 +184,22 @@ SDL_GpuDevice *SDL_GpuCreateDevice(
     return result;
 }
 
+SDL_GpuDevice *SDL_GpuCreateDevice(
+    SDL_bool debugMode,
+    SDL_bool preferLowPower,
+    const char *name)
+{
+    return SDL_GpuCreateDeviceCommon(debugMode, preferLowPower, name, 0);
```

Comment:
Typically this is done by having the simple version call *WithProperties() with properties of 0, to get default values, and then all the logic in CreateDeviceCommon() would be in CreateDeviceWithProperties()

Reply:
This will get done in the next PR for CreateDevice - we need to add one more parameter before making the internal change.

For bitflags, is there a preferred way to do this? I was thinking about a list, formatted similar to the videodriver/audiodriver hints.

Reply:
For windows there's a separate property for each bit, to make it more extensible.

Reply:
Makes sense - this change should be in tomorrow's weekly update!